templates/default/config/siteConfig.js

```diff
@@ -3,6 +3,9 @@ import userConfig from '../content/config.js';
 const defaultConfig = {
   title: 'Flowershow',
   description: 'Publish your digital garden',
+  repoRoot: "https://github.com/flowershow/flowershow",
+  repoEditPath: "/edit/main/site/content/",
+  editLinkShow: false,
```

Comment:
We actually do want to have it set to `true` by default :) I'd just turn it off for `index.md` page (explicitly, in frontmatter) as it won't look good on the home page imo.